doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
 
-Problem: Increasing number of kernels 
+
+Problem 3: Increasing the number of kernels 
 -------------------------------------
 
-Kernel libraries need to be integrated with multiple deep learning frameworks, 
-and this arduous task becomes even harder due to increased numbers of required 
-kernels for achieving optimal performance. The number of required kernels is 
-product of number of chip designs, data types, operations, and the cardinality 
-of each parameter for each operation. In the past, the number of required 
-kernels was limited, but as the AI research and industry rapidly develops, the 
-final product of required kernels is increasing exponentially. 
+As previously mentioned, Kernel libraries need to be integrated with multiple deep learning frameworks, and this already arduous task becomes even harder due to the greater number of a kernels for achieving optimal performance. The number of required kernels is based on the number of chip designs, data types, operations, and the cardinality of each parameter per operation. In the past, the number of required kernels was limited, but as AI research and industry continue to rapidly develop, the number of required kernels is exponentially increasing. 
 
-.. _figure-B:
+.. _figure-C:
 
 .. figure:: ../graphics/intro_kernel_explosion.png
    :width: 555px
    :alt: 
 
-   Each of these connections represents significant work for what will 
-   ultimately be a brittle setup that is enormously expensive to maintain.
+   Each of these connections represents significant work for what will ultimately be a fragile setup that is costly to maintain.
 
 
+PlaidML addresses the exponential growth of required kernels. It takes two inputs: the operation defined by the user and the machine description of the hardware target. 
 
-PlaidML addresses the kernel explosion problem in a manner that lifts a heavy 
-burden off kernel developers. It automatically lowers networks from nGraph 
-into Tile, a :abbr:Domain-Specific Language (DSL) designed for deep learning 
-that allows developers to express how an operation should calculate tensors in
-an intuitive, mathematical form via `Stripe`_. Integration of PlaidML with 
-nGraph means extra flexibility to support newer deep learning models in the 
-absence of by-hand optimized kernels for the new operations.
-
+PlaidML uses Tile, a :abbr:Domain-Specific Language (DSL) that allows developers to express how an operation should calculate tensors in an intuitive, mathematical form. PlaidML takes user-defined Tile code along with the machine description (threads, max memory input, etc.) to automatically apply low-level optimizations. These automated optimizations do not require kernel developers to write kernel libraries, easing their burden. Integrating PlaidML with nGraph provides flexbility to support newer deep learning models in the absence of hand-optimized kernels for new operations.
```

Comment:
PlaidML is actually moving in the direction of `Stripe` and away from `Tile`. While this is accurate pertaining to the mechanics of `Tile`, we would like to emphasize in the high-level description of PlaidML that `Stripe` is our intermediate representation (IR) that utilizes iterative optimization passes.

Please contact me directly so that we can work together on adding a better summary of `Stripe`.